torch/onnx/utils.py

```diff
@@ -705,12 +708,12 @@ def _export(model, args, f, export_params=True, verbose=False, training=None,
             if export_params:
                 proto, export_map = graph._export_onnx(
                     params_dict, opset_version, dynamic_axes, defer_weight_export,
-                    operator_export_type, strip_doc_string, val_keep_init_as_ip, custom_opsets,
```

Comment:
Is the strip_doc_string (not verbose) still needed inside the _export_onnx function?

Reply:
Yes, we just replace strip_doc_string with verbose(with an opposite meaning). User still have a chance to decide if those information will be left in ONNX graph or not.